ring/client/pool.go

```diff
@@ -171,7 +171,7 @@ func (p *Pool) removeStaleClients() {
 	}
 
 	for _, addr := range p.RegisteredAddresses() {
-		if util.StringsContain(serviceAddrs, addr) {
+		if stringsutil.SliceContains(serviceAddrs, addr) {
```

Comment:
This can use `slices.Contains()` since we target go 1.18

Reply:
Nice! I didn't know.